### DISCUSSION TITLE
How do I check my Docksal version?  (And how do I update it?!)

### DISCUSSION THREAD

**Discussion by dpremo-ul** [docksal/docksal#1561] [Q&A] [answered]
When I run fin --version ir shows me a number like 1.106.0.  How do I check the version of Docksal I'm running?  

It seems like this information should be included with every release.  And how do I update Docksal?  I can't find a command to do it. 

Do I have to update the image I'm using?  I know how to do that.  If that's it, please make it obvious.  

It seems like this information should be displayed prominently.

**Comment by lpeabody** [comment, marked as answer]
You can always update Docksal to latest released version using `fin update` https://docs.docksal.io/getting-started/setup/#updates. You can also update or downgrade to a specific version by running `DOCKSAL_VERSION=develop fin update`, which would update you to the version at the tip of develop in the `docksal/docksal` repository.

Regarding the overall version of Docksal, currently the best way to identify the current version you're on is to get the version of the `fin` script (`fin version`) and then cross-reference that with the release it was assigned to in the Releases section of this repository. I think there is room for enhancement here. @lmakarov what are your thoughts in including the overall Docksal version in the `fin version` info?

**Reply by lmakarov** [reply]
> @lmakarov what are your thoughts in including the overall Docksal version in the fin version info?

This would definitely help with the confusion between the overall Docksal version and fin version.

The reason why we avoided this back in the days was the ambiguity of what the Docksal version would be in-between two releases. If we call it version `next` or just `dev` / `develop`, then we'd need to change it back and forth during and after each release. That seemed like an unnecessary overhead (even though the effort is minimal).

A simpler approach to that would be to just show the "most recent" Docksal release + the current version of fin.

Created a ticket: https://github.com/docksal/docksal/issues/1574.
